mirascope/prompts.py

```diff
@@ -6,10 +6,12 @@
 import re
 from string import Formatter
 from textwrap import dedent
-from typing import Type, TypeVar
+from typing import Callable, Optional, Type, TypeVar, Union
 
 from pydantic import BaseModel
 
+from mirascope.chat.types import OpenAIChatCompletion
```

Comment:
I'm realizing that this adds `openai` as a required dependency of `mirascope`.

Seems better for the prompt to just accept the dict and the user passes in the completion dump?

Reply:
hmmm